core/src/autogluon/core/scheduler/seq_scheduler.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import time
+from collections import OrderedDict
+from copy import deepcopy
+
+from autogluon.core.searcher import GPFIFOSearcher
+from autogluon.core.utils import EasyDict
+from tqdm.auto import tqdm
+
+logger = logging.getLogger(__name__)
+
+
+class LocalReporter:
+    def __init__(self, trial, config, training_history: dict, config_history: dict):
+        self.trial = trial
+
+        self.training_history = training_history
+        if trial not in self.training_history:
+            self.training_history[trial] = []
+
+        self.task_config = EasyDict(deepcopy(config))
```

Comment:
You pass train_fn.kwvars when calling this, which does not know about searcher_config. Something is wrong here ...

Reply:
Yes, there was a defect here: it supposed to be `searcher_config = self.searcher.get_config()` - already fixed it.